meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-ti/secdev/ti-k3-secdev_git.bb

```diff
@@ -0,0 +1,40 @@
+DESCRIPTION = "Security development tools for High-Security(HS) TI K3 processors."
+HOMEPAGE = "https://git.ti.com/cgit/security-development-tools/core-secdev-k3"
+SECTION = "devel"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://manifest/k3-secdev-0.2-manifest.html;md5=f632a78870cc64550078d7b3cbac0892"
+
+# set a default value for TI_K3_SECDEV_INSTALL_DIR_RECIPE
+export TI_K3_SECDEV_INSTALL_DIR_RECIPE = "${datadir}/ti/ti-k3-secdev"
```

Comment:
Do we need to export this variable here? I think it is only used in do_install.

Reply:
Without ti-extras layer we don't have the `ti-linux-fw.inc` where this variable is defined and exported.
In the upstream patch discussion someone from TI suggest to remove the `ti-linux-fw.inc` and replace
all tese variables with the default values. That is something I will do in a follow-up patch.
Like with the `ti-sci-fw_git.bb` a bbappend also work here.

Reply:
Oh sure, my comment was more if defining with `TI_K3_SECDEV_INSTALL_DIR_RECIPE = "${datadir}/ti/ti-k3-secdev`, without export, would already be enough (since the scope of this variable seems to be only in do_install).

Here we have to duplicate the recipe because an append would not work since the original recipe is not yet in meta-ti-bsp.

Reply:
I will leave the export like the definition on `ti-paths.inc`
```
grep TI_K3_SECDEV_INSTALL_DIR_RECIPE meta-ti-extras/recipes-ti/includes/ti-paths.inc
export TI_K3_SECDEV_INSTALL_DIR_RECIPE = "${installdir}/ti-k3-secdev"

```

Reply:
Sure, that's fine.